lib/article_status_manager.rb

```diff
@@ -45,6 +46,7 @@ def update_status(articles)
 
     # First we find any pages that just moved, and update title and namespace.
     update_title_and_namespace @synced_articles
+    ArticleChangeNamespaceAlertManager.create_alerts_for_article_namespace_change
```

Comment:
The idea here is to only do this for the specific articles where we needed to change the namespace. It looks like the current version of this PR doesn't account for detecting a namespace change.

Reply:
Oh yeah. Is it advisable to do that in the alert manager or here in the ArticleStatusManager?

Reply:
Like this recent change?

Reply:
Oh okay. Lemme update